src/cctag/Identification.cpp

```diff
@@ -260,8 +260,8 @@ void extractSignalUsingHomography(
   {
     applyHomography(xRes, yRes, mHomography, x, y);
 
-    if ( xRes >= 1.f && xRes <= src.cols-1 &&
-         yRes >= 1.f && yRes <= src.rows-1 )
+    if ( xRes >= 0.f && xRes < src.cols-1 &&
```

Comment:
either `>0` or `>=1` i think.
Since the other condition excludes the last pixel, I guess also the first pixel should be excluded (to take into account border pixels, i guess) as it was originally.

Reply:
The interpolation inside access to `array[int(x)+0]` and `array[int(x)+1]` and interpolate between the 2. So it's not symmetric, as we can retrieve the value on `(0,0)` but not the value on `(width-1, 0)`.